containter_host.tf

```diff
@@ -58,6 +58,11 @@ resource "null_resource" "cluster" {
     destination = "/home/${var.user}/bootstrap-cluster.sh"
   }
 
```

Comment:
won't this compel some one who does not want monitoring to provision for one?

Reply:
I say "not wanting monitoring" should not even be an option ;) So if this makes you get monitoring anyway, I'd say that's fine ;)

Reply:
All right. I'm ok.  @loafoe ??